test/units/testsuite-49.sh

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env bash
+# -*- mode: shell-script; indent-tabs-mode: nil; sh-basic-offset: 4; -*-
+# ex: ts=8 sw=4 sts=4 et filetype=sh
+set -ex
+set -o pipefail
+
+modprobe loop
```

Comment:
this should be unnecessary

Reply:
I'll check again, might be leftover of putting the image together with all the extra modules